packages/uilib/src/lib/plugins/plugin-store/plugin-store.svelte

```diff
@@ -0,0 +1,227 @@
+			(key, value) => value
+		)
+	)
+}
+
+function withoutContent<P extends Plugin>(plugin: P): P {
+	return { ...plugin, content: undefined }
+}
+
+function storePlugins(plugins: Array<Plugin>) {
+	localStorage.setItem('plugins', JSON.stringify(plugins.map(withoutContent)))
+}
+
+// Enables/disables plugin by toggling the "installed" property.
+function togglePluginState(plugin: Plugin, isEnabled: boolean) {
+	const currentPlugins = storedPlugins()
+	currentPlugins.find((it) => it.name === plugin.name).installed = isEnabled
+	storePlugins(currentPlugins)
+	plugins = currentPlugins
+	plugin.installed = isEnabled
+	notificationSnackbar.open()
+}
+
+// Completely removes plugin from local browser cache.
+function uninstallPlugin(plugin: Plugin) {
+	const currentPlugins = storedPlugins()
+	const updatedPlugins = currentPlugins.filter(
+		(it) => it.name !== plugin.name
+	)
+	storePlugins(updatedPlugins)
+	plugins = updatedPlugins
+}
+
+let plugins = storedPlugins()
+
+let showOnlyInstalled = false
+let searchFilter = ''
+
+$: filteredPlugins = plugins
+	.filter(
+		(plugin) => !showOnlyInstalled || plugin.installed === showOnlyInstalled
+	)
+	.filter((plugin) =>
+		plugin.name.toLowerCase().includes(searchFilter.toLowerCase())
+	)
+	.filter((plugin) => plugin.name !== 'PluginStore')
+	.filter((plugin) => plugin.name !== 'Plugin Store')
+
+// #endregion
+
+// #region UI
+let notificationSnackbar: Snackbar
+
+let currentPluginAnchor: Element
+let menus: Menu[]
+$: menus = filteredPlugins.map(() => null)
+$: menuStates = filteredPlugins.map(() => false)
+
+function openPluginMenu(e, index: number) {
+	currentPluginAnchor = e.currentTarget
+	menuStates = menuStates.map(() => false)
+	menuStates[index] = true
+}
+
+// #endregion
+</script>
+
+<Dialog bind:open={isOpen} 
+fullscreen
+surface$style="width: 850px; max-width: calc(100vw - 32px);"
+aria-labelledby="plugin-store-title" 
+aria-describedby="plugin-store-content">
+<Header>
+    <Title id="plugin-store-title">Plugin Store</Title>
+    <IconButton class="material-icons" action="close">
+        <IconClose/>
+    </IconButton>
+</Header>
+<Content id="plugin-store-content">
+<Theme>
+    <plugin-store>
+        <plugin-store-toolbar>
+            <Checkbox label="Only Installed" bind:checked={showOnlyInstalled}/>
+            <Textfield
+            label={"Search"}
+            variant={"outlined"}
+            bind:value={searchFilter}
+            />
+        </plugin-store-toolbar>
+        <plugin-store-items> 
+            {#each filteredPlugins as plugin, index}
+            <plugin-store-item style={`background: rgba(0, 0, 0, ${index % 2 === 0 ? "0" : "0.03"});`}>
+                <plugin-store-item-meta>
+                <span style="font-size: 0.85rem;">Author</span>
+                <span><strong>{plugin.name}</strong></span>
+                <span style="font-size: 0.85rem;">https://sprinteins.github.io/plugins/{plugin.name.split(" ").join("-").toLowerCase()}/index.js</span>
```

Comment:
Not familiar with the concept, but i would just show the "local" link (eg. substation, template-generator usw.) and add a "copy" button to the right hand side to copy the full URL to the clipboard. Just a suggestion from my side, lmk if you can play around with this or its against the concept

Reply:
Thanks for the suggestion, I'll keep in contact with Tamas about this and will implement it, if approved. 
The link is actually in this format in the UI mockup that I got currently it doesn't exist. Again, this is a placeholder as well just like the author text.